memeid4s-http4s/src/main/scala/memeid4s/http4s/instances.scala

```diff
@@ -36,7 +34,7 @@ trait instances {
 
   /** Allow using UUIDs in a request's query params */
   implicit val UUIDQueryParamEncoderInstance: QueryParamEncoder[UUID] =
-    QueryParamEncoder[String].contramap(_.show)
+    QueryParamEncoder[String].contramap(_.toString)
```

Comment:
out of curiosity, why was this change necessary?

Reply:
To be able to not depend on `memeid-cats`

Reply:
ah makes sense, thanks :+1: